web/src/pages/CreateRoom/CreateRoom.tsx

```diff
@@ -0,0 +1,111 @@
+import { Component, createSignal, Show } from "solid-js";
+import Button from "@/components/Button";
+
+import styles from "./CreateRoom.module.scss";
+import useCreateRoom from "./hooks/useCreateRoom";
+
+const prohibitedEmojiHex = "0x1F6AB";
+
+const CreateRoom: Component = () => {
+	const { fields, updateField, submit } = useCreateRoom();
+
+	const handleSubmit = (event: Event): void => {
+		event.preventDefault();
+		submit();
+	};
+
+	return (
+		<main class={styles.createRoom}>
+			<form class={styles.createRoomForm} onSubmit={handleSubmit}>
+				<label for="moderatorName">
+					Your name
+					<span class={styles.required}>*</span>
+				</label>
+
+				<input
+					name="moderatorName"
+					required
+					type="text"
+					onInput={updateField}
+				/>
+
+				<div class={styles.seperator}>
+					<hr />
+				</div>
+
+				<label for="voterOptions">
+					What options would you like voters have to choose from?
```

Comment:
Grammatically this is weird.  It feels like you are missing a "to"

Maybe 
"What options would you like voters to choose from?"  hanging preposition :P 
"From what options would you like voters to choose?" grammatically correct but sounds terrible
"What options would you like voters to have to choose from" I think that's what you are going for